streaming/src/main/scala/org/apache/spark/streaming/Checkpoint.scala

```diff
@@ -44,11 +44,19 @@ class Checkpoint(@transient ssc: StreamingContext, val checkpointTime: Time)
   val sparkConfPairs = ssc.conf.getAll
 
   def createSparkConf(): SparkConf = {
+    val reloadConfs = List(
```

Comment:
can you call this `propertiesToReload`? It's confusing to have `reloadConfs` here and ` newReloadConf` down there when they're not describing the same thing.